src/Sarif/Baseline/V2/WhatComponent.cs

```diff
@@ -42,19 +42,30 @@ public override bool Equals(object obj)
 
         public override int GetHashCode()
         {
-            int hashCode = 0;
+            int hashCode = 17;
```

Comment:
>17 [](start = 27, length = 2)

Reimplemented to match the (generated) `GetHashCode` implementations in the OM classes.